workerapi/server/setup.go

```diff
@@ -43,19 +47,27 @@ func Defaults() (*ice.MagicBag, jsonconfig.Schema) {
 		endpoints.MakeStatsReceiver,
```

Comment:
nope i really like it too!  Makes it much more readable in my opinion,  I can update the rest of the functions to match!